lib/rialto/etl/configs/wos_to_sparql_statements.rb

```diff
@@ -54,8 +54,9 @@ def parse_address(addr)
 # Find the grant agencies
 def fetch_grant_agencies(json)
   agencies = Array.wrap(JsonPath.on(json, '$.static_data.fullrecord_metadata.fund_ack.grants.grant').first).map do |grant|
-    next if grant.fetch('grant_agency', []).empty?
-    grant['grant_agency']
+    grant_agency = grant['grant_agency']
```

Comment:
I believe we have ActiveSupport required in this context, so we could make use of `#blank?` and reduce this to:

```ruby
next if grant['grant_agency'].blank?
```

What's here now is a-ok, too.